### PR TITLE
Convert `admin/webhooks` spec controller->system

### PR DESCRIPTION
**Pull request by mjankowski**
Conversion pretty simple.

Possible future spec refactor here ... the interaction between "an admin user" and the default setup for a webhook fabricator is a little awkward. I had to do a specific event value instead of the fabricator default to have this work as expected, I assume from a policy thing. Might be worth updating fabricator such that default allows admin user to manage them.

### CODE REVIEW

File: spec/system/admin/webhooks_spec.rb [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+        click_on I18n.t('admin.webhooks.add_new')
+        expect(page)
+          .to have_content(I18n.t('admin.webhooks.new'))
+
+        fill_in 'webhook_url', with: ''
+        expect { submit_form }
+          .to_not change(Webhook, :count)
+        expect(page)
+          .to have_content(/errors below/)
```
Comment (ClearlyClaire) [inline spec/system/admin/webhooks_spec.rb:37]: It's a minor thing, but we're not checking empty events anywhere.

Comment (ClearlyClaire) [reply spec/system/admin/webhooks_spec.rb:37]: Oh yeah, it's testing empty URL and empty events at the same time. Maybe it should be split in two, and made a little more explicit with comments.

Comment (mjankowski) [reply spec/system/admin/webhooks_spec.rb:39]: Added some scattered comments to add some context about what's being exercised in a few spots.

Comment (mjankowski) [reply spec/system/admin/webhooks_spec.rb:39]: Will do ... but out of curiosity (mostly to inform future PRs along these lines) ... why?

Generally speaking I'd try to be exhaustive on all combinations of how we can make a thing invalid at the model spec level; and then at any of the controller/system/request spec level, just make sure we hit the "invalid path" once to assert that some errors showed up and the invalid attrs did not save a record (exception here would be if there were multiple ways to be invalid, multiple filters, other edge cases, etc - to exercise them all) ... but in a way where it sort of doesnt matter which particular combination of making it invalid we used.

Again, no objection, just want to understand rationale for future of these. Will update this.

Comment (mjankowski) [reply spec/system/admin/webhooks_spec.rb:39]: Added special case for no events selected.

**Review by ClearlyClaire — APPROVED** [review]

File: spec/system/admin/webhooks_spec.rb (hunk 1 above)
Comment (ClearlyClaire) [reply spec/system/admin/webhooks_spec.rb:39]: Yeah, I suppose you're right, if the model tests cover this, it does not matter much.

**Pull request merged**